src/deluge/gui/ui/menus.cpp

```diff
@@ -456,10 +427,12 @@ reverb::RoomSize reverbRoomSizeMenu{STRING_FOR_ROOM_SIZE};
 reverb::Dampening reverbDampeningMenu{STRING_FOR_DAMPENING};
 reverb::Width reverbWidthMenu{STRING_FOR_WIDTH, STRING_FOR_REVERB_WIDTH};
 reverb::Pan reverbPanMenu{STRING_FOR_PAN, STRING_FOR_REVERB_PAN};
+reverb::Model reverbModelMenu{STRING_FOR_MODEL};
 
 Submenu reverbMenu{
     STRING_FOR_REVERB,
     {
+        &reverbModelMenu,
```

Comment:
Can you add &reverbModelMenu to the `Submenu globalReverbMenu{` below as well :)? Thx!

Reply:
Done.